### PR TITLE
fix: skip optimizing when running with --debug

### LINKED ISSUE #44: Skip optimizing when running with --debug
I'm seeing `Optimizing .wasm file...` in the output when running with `--debug`

### PR DESCRIPTION
**Pull request by nissy-dev**
fix #44 

Based on #44, I fixed optimization is no longer performed with debug flag.

### CODE REVIEW

**Comment by nissy-dev** [comment]
@dsherret if you have time, can you review my pr?

File: lib/commands/build_command.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export async function runBuildCommand(args: BuildCommand) {
   if (output.wasmFileName != null) {
     const wasmDest = path.join(args.outDir, output.wasmFileName);
     await Deno.writeFile(wasmDest, new Uint8Array(output.bindgen.wasmBytes));
-    if (args.isOpt) {
+    if (args.profile === "release" && args.isOpt) {
```
Comment (dsherret) [inline lib/commands/build_command.ts:20]: Small point, but we should probably move this check to here and have it check `flags.debug`:

https://github.com/denoland/wasmbuild/blob/e1fd6ce68aedb15fcd99ff139e0307faa409ca66/lib/args.ts#L59

Comment (nissy-dev) [reply lib/commands/build_command.ts]: Thank you for comments! I refactored in 31bb43d

**Review by dsherret — APPROVED** [review]
LGTM. Thanks @nissy-dev!

**Pull request merged**